### ISSUE TITLE
usermanager load error on Safari / macOS

### ISSUE THREAD

**Issue by duncdrum** [eXist-db/usermanager#11]
There seems to be a problem during the compile of usermanager, map files are missing which causes Safari (regular and tech preview) to not display anything after log-in, Chrome and Firefox are fine:

![screen shot 2018-02-14 at 18 35 41](https://user-images.githubusercontent.com/6205362/36219205-dd93c210-11b6-11e8-8cca-c3d74013ec17.png)

console errors:
```
[Error] Failed to load resource: the server responded with a status of 404 (Document /db/apps/usermanager/apply-shim.min.js.map not found) (apply-shim.min.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Document /db/apps/usermanager/custom-style-interface.min.js.map not found) (custom-style-interface.min.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Document /db/apps/usermanager/webcomponents-lite.js.map not found) (webcomponents-lite.js.map, line 0)
```

Usermanager: `0.15.1`
eXist: `4.0.0`
dashboard: `1.0.0`
macOS: `10.13.3`
Safari: `11.0.3`
Firefox: `58.0.2` 
Chrome :`64.0.3282.167`

**Comment by JoernT** [comment]
yeah - this looks like an build optimizer issue. Will look into it.

**Comment by dizzzz** [comment]
@JoernT any update? this is rather an urgent issue; on safari the app is not functional at all, with Chrome seem to look
<img width="1027" alt="existdb-usermanager_and_inbox__13_messages__3_unread_" src="https://user-images.githubusercontent.com/1700062/36352242-17bf633e-14b6-11e8-90e7-be4ad4162db6.png">
 (...) ok

**Comment by JoernT** [comment]
@dannes why do you think it's that high priority? Chrome and FF should work fine so at least no road-block for people i guess.

Still investigated that issue but it's extremely strange cause it's working under certain conditions (empty cache, freshly started Safari, fresh login). That's at least what i found so far. Will keep on looking into it.

**Comment by JoernT** [comment]
more detailed steps to reproduce:
* start a fresh safari
* clean the cache
* load the usermanager
-> App will be loaded correctly

Once you reload it stops working.

**Comment by dizzzz** [comment]
Joern, I typically only use one browser, and that one was failing :-/ when I checked the details in the browser  I saw that these 3 files were missing. At that moment i really really needed to have access the app to change modifications. my workaround was: stop database, run client.sh in local mode and do the required user changes. As I did not know about the other browsers were (in a way?) functional, I marked it as 'hi prior' because it blocking. If it has offended you, sorry :-/

**Comment by JoernT** [comment]
@dizzzz no problem - it wasn't working in Safari but that's fixed now. However just to inform you: Chrome is most reliable with web components as it implements the current spec natively already; Safari has incomplete native support and Firefox also. The latter two are polyfilled by Polymer.

Usermanager should indeed work. If issues come up please report. 

Issue should be fixed in 0.16.0 now but as i found further bugs i quickly pushed out 0.16.2 which addresses further small issues. The latter one will go into public repo (unless further testing doesn't say otherwise ;)

**Comment by dizzzz** [comment]
I was not aware that web component support is -at this moment- nod widely there; I guess MSIE/edge is not much better.

On MacOS I use safari only as this browser has so little impact on battery life. Chrome is a problem here :-(
